### DISCUSSION TITLE
Will Ubuntu 22.04 image update it's chrome browser to version 109 anytime soon?

### DISCUSSION THREAD

**Discussion by IanCTH** [actions/virtual-environments#6917] [Installed software]
Currently the[ Ubuntu 22.04 image ](https://github.com/actions/runner-images/blob/ubuntu22/20230109.1/images/linux/Ubuntu2204-Readme.md) uses the chrome browser version 108.0.5359.124. 

We've recently updated our npm package to use `"chromedriver": "^109.0.0"` and now our tests are failing with the following error:
```
Test Suites: 1 skipped, 0 of 1 total
Tests:       13 skipped, 13 total
Snapshots:   0 total
Time:        8.23[5](https://github.com/.../.../actions/runs/3906912870/jobs/6675748114#step:6:6) s
Ran all test suites matching /resourceLibrary.test/i.
Jest did not exit one second after the test run has completed.

This usually means that there are asynchronous operations that weren't stopped in your tests. Consider running Jest with `--detectOpenHandles` to troubleshoot this issue.
/home/runner/work/.../.../node_modules/selenium-webdriver/lib/promise.js:2[6](https://github.com/.../.../actions/runs/3906912870/jobs/6675748114#step:6:7)26
        throw error;
        ^

SessionNotCreatedError: session not created: This version of ChromeDriver only supports Chrome version 109
Current browser version 108.0.5359.124 with binary path /usr/bin/google-chrome
```

Will Ubuntu image 22.04 update it's chrome browser to version 109 anytime soon?

Thanks!

P.S. A quick fix for anyone encountering the same problem: I run the following script in the running instance of the image. It simply downloads the chrome browser and installs it, replacing the default chrome version `108` with the newly installed version `109`

```bash
wget -q -O /tmp/chrome.deb http://dl.google.com/linux/chrome/deb/pool/main/g/google-chrome-stable/google-chrome-stable_109.0.5414.74-1_amd64.deb \
  && sudo apt install -y /tmp/chrome.deb \
  && rm /tmp/chrome.deb
```

**Comment by mikhailkoliada** [comment]
Hello, we always install the lates version of the chrome, so it should be available next week
